src/main/kotlin/no/nav/familie/ks/sak/api/BehandlingStegController.kt

```diff
@@ -41,7 +41,7 @@ class BehandlingStegController(
     }
 
     @PostMapping(path = ["vilkårsvurdering"])
-    fun validerVilkårsvurdering(@PathVariable behandlingId: Long): ResponseEntity<Ressurs<BehandlingResponsDto>> {
+    fun utførVilkårsvurdering(@PathVariable behandlingId: Long): ResponseEntity<Ressurs<BehandlingResponsDto>> {
         tilgangService.validerTilgangTilHandling(
             minimumBehandlerRolle = BehandlerRolle.SAKSBEHANDLER,
             handling = "vurdere vilkårsvurdering"
```

Comment:
```suggestion
            handling = "utfør vilkårsvurdering"
```
1-1 handling og metodenavn